lib/manager/gradle/index.js

```diff
@@ -25,125 +25,56 @@ async function extractAllPackageFiles(config, packageFiles) {
     }
   }
 
-  await configureUseLatestVersionPlugin(config.localDir);
+  await updatesReport.createRenovateGradlePlugin(config.localDir);
   const gradleSuccess = await executeGradle(config);
   if (!gradleSuccess) {
+    logger.warn('No gradle dependencies found');
     return null;
   }
 
   gradle.init();
+
+  const dependencies = await updatesReport.extractDependenciesFromUpdatesReport(
+    config.localDir
+  );
+  if (dependencies.length === 0) {
+    return [];
+  }
+
   const gradleFiles = [];
   for (const packageFile of packageFiles) {
     const content = await platform.getFile(packageFile);
     if (content) {
-      const deps = await extractPackageFile(content, packageFile, config);
-      if (deps) {
-        gradleFiles.push({
-          packageFile,
-          manager: 'gradle',
-          ...deps,
-        });
-      }
+      gradleFiles.push({
```

Comment:
had to add all the dependencies to all the files because I don't have an easy way to differentiate which dependencies goes with which file, and not sure if really matters, probably it's easier just try to see if it's there